app/src/main/res/layout/dashboard_funding_view.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  android:id="@+id/creator_dashboard"
+  android:layout_height="match_parent"
+  android:layout_width="match_parent"
+  android:orientation="vertical"
+  xmlns:tools="http://schemas.android.com/tools">
+
+  <TextView
+    android:id="@+id/creator_dashboard_project_name"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingTop="@dimen/grid_new_2"
+    android:paddingStart="@dimen/grid_new_2"
```

Comment:
we should add a `android:paddingEnd="@dimen/grid_new_2"` here too for long project names, so there's even spacing on both ends